contrib/babelfishpg_common/src/typecode.c

```diff
@@ -200,6 +200,9 @@ typecode_list(PG_FUNCTION_ARGS)
 		Datum		values[7];
 		bool		nulls[7];
 
+		if(strcmp(info->pg_typname,"rowversion") == 0)
+		continue;
```

Comment:
Please explain why you chose to remove duplicate entry here rather than removing it from type_infos itself?

Reply:
I actually did that change ( removing rowversion datatype) in type_infos array , but that was causing failure in bcp query in dotnet test framework.